lib/update-bridgehead.sh

```diff
@@ -50,12 +50,11 @@ for DIR in /etc/bridgehead $(pwd); do
     git -C $DIR config credential.helper "$CREDHELPER"
   fi
   old_git_hash="$(git -C $DIR rev-parse --verify HEAD)"
-  if [ -z "$HTTP_PROXY_URL" ]; then
-    log "INFO" "Git is using no proxy!"
```

Comment:
Why did you remove that?

Reply:
Thanks, fixed in 9fc8564e4ef280403deb472a40eda255e02d740e